scala.html.markdown

```diff
@@ -12,22 +12,62 @@ Scala - the scalable language
 
 ```scala
 
+/////////////////////////////////////////////////
+// 0. Basics
```

Comment:
Hmm, maybe call this section "Setting up"? We can also combine the two chunks of comments into one.